src/modules/mentornity/src/view-wrappers/mentornity-member-home-widgets-wrapper/mentornity-member-home-widgets-wrapper.tsx

```diff
@@ -0,0 +1,136 @@
+
+@Component({
+  tag: 'mentornity-member-home-widgets-wrapper',
+})
+export class MentornityMemberHomeWidgetsWrapper {
+
+  @State() error?: ApiError;
+  @State() meetingGoalConfig?: MeetingGoalConfig;
+  @State() pairMeetingSummary?: { user: User, meetings: Event }[];
+  @State() atamaMemberPairsOverviewViewConfig?: AtamaMemberPairsOverviewViewConfig;
+  @State() organisationConfig?: Config;
+  @State() params?: any;
+
+  @Prop() language: string;
+  @Prop() userType: string;
+  @Prop() menteeType: string;
+  @Prop() groupId: string;  // TODO use for grouped programs
+
+  private mentornityBridgeApi: MentornityBridgeApi;
+
+  @Listen('action')
+  onAction(event: CustomEvent<NavigationAction>) {
+    const action = event.detail;
+
+    switch (action.name) {
+      case 'go-to-pairs-page':
+        // dispatch event for Mentornity web-client to handle the action
+        window.dispatchEvent(new CustomEvent<any>(
+          'dvn.mentornity-user-card-list-wrapper.open-pairs-page',
+        ));
+        break;
+      case 'go-to-user-pair-page':
+      // dispatch event for Mentornity web-client to handle the action
+      window.dispatchEvent(new CustomEvent<any>(
+        'dvn.mentornity-user-card-list-wrapper.open-user-pair-page', {
+          detail: {
+            userId: action.params.uid,
+            subId: action.params.subId,
+          }
+        },
+      ));
+      break;
+      case 'go-to-mentor-selection':
+      // dispatch event for Mentornity web-client to handle the action
+      window.dispatchEvent(new CustomEvent<any>(
+        'dvn.mentornity-user-card-list-wrapper.open-mentor-selection-page',
+      ));
+      break;
+    }
+  }
+
+  async componentWillLoad() {
+    const { language } = this;
+    const { context, headers } = getSessionDetails(this.groupId);
+    
+    const mentornityApi = createMentornityApiInstance(mentornityApiEndpoint, headers);
+
+    // atama api
+    this.mentornityBridgeApi = new MentornityBridgeApi(mentornityApiEndpoint, context, headers);
+
+    const organisationConfig = await mentornityApi.getOrganisationConfig(context.organisationId);
+    this.organisationConfig = organisationConfig;
+
+    this.meetingGoalConfig = await mentornityApi.getOrganisationMeetingGoalConfig(context.organisationDomain)
+      .catch(error => this.error = error);
+
+    const getPairs = context.role === 'mentor' ? this.menteeType === 'groups' ? mentornityApi.getGroups : mentornityApi.getMentees : mentornityApi.getMentors;
+    const pairs = await getPairs(context.organisationId)
+      .catch(error => this.error = error);
+
+    const memberRolePath = context.role === 'mentor' ? 'mentors' : 'mentees';
+    const meetings = await mentornityApi.getMeetings(context.organisationId, memberRolePath, context.userId, {
+      status: 'active',
+    }).catch(error => this.error = error);
+
+    const pairMeetingSummary = [];
+    pairs.results
+      .filter(pairedUser => pairedUser.subscription?.status === 'active')
```

Comment:
Üst tarafta yazdığımı yapınca pairs yerine activePairs kullanabilirsin.
```suggestion
    activePairs
```